__tests__/Container.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import {decorators} from '../src';
+import Container from '../src/components/Decorators/Container';
+import animations from '../src/themes/animations';
+import data from '../example/data';
+import style from '../src/themes/default';
+
+const onClick = jest.fn();
+
+const renderComponent = (props = {}) => <Container
+    {...{decorators, onClick, animations}}
+    node={data}
+    style={style.tree.node}
+    {...props}
+/>;
+
+describe('<Container/>', () => {
+    it('should return children with contains decorators.Header if terminal prop is true', () => {
```

Comment:
the correct sentence is: `should contains a decorators.Header into their children`